tests/SpeechConfigTests.ts

```diff
@@ -877,6 +877,40 @@ describe("Connection URL Tests", (): void => {
                     );
                 });
 
+                test("segmentationMaximumTimeMs", (done: jest.DoneCallback): void => {
+                    // eslint-disable-next-line no-console
+                    console.info("Name: segmentationMaximumTimeMs");
+
+                    const propName: string = "segmentationMaximumTimeMs";
+                    const val: string = "1004";
+
+                    testUrlParameter(createMethod,
+                        (s: sdk.SpeechConfig): void => {
+                            s.setProperty(sdk.PropertyId[sdk.PropertyId.Speech_SegmentationMaximumTimeMs], val);
+                        },
+                        recognizerCreateMethod,
+                        done,
+                        propName + "=" + val
+                    );
+                });
+                
+                test("segmentationStrategy", (done: jest.DoneCallback): void => {
```

Comment:
This is winding up in the URL instead of just in the context message?

Reply:
Yes, I found existing slienceTimeOut in query url and thus I also added maxSegmentTimeout and strategy to url